spec/models/spotlight/resources/purl_spec.rb

```diff
@@ -59,7 +62,9 @@
     end
 
     it "should add a document to solr" do
-      expect(subject).to receive(:update_index)
+      solr_data = '[{"spotlight_resource_id_ssim":null,"spotlight_resource_type_ssim":"spotlight/resources/purls","upstream":true}]'
+      expect(blacklight_solr).to receive(:update).with({params: {commitWithin: 500}, data: solr_data, headers: {"Content-Type" => "application/json"}})
```

Comment:
I'm surprised Rubocop isn't complaining about the lack of whitespace around the braces.